services/tracks/src/services/genresService.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const { v4: uuid } = require('uuid');
+const { Genres } = require('../db/models/Genres');
+const { errorMessages } = require('../config');
+
+class GenreService {
+  async getGenres() {
+    const genres = await Genres.findAll({ order: ['id'] });
+    return genres;
```

Comment:
I think you should return it immediately. In any case, the asynchronous function will return a promise, so there is no need to wait for the result of the service Genres
```suggestion
    return Genres.findAll({ order: ['id'] });
```